R/app_template.R

```diff
@@ -38,7 +38,7 @@
 #' * `app.R` is the main application file.
 #' * All files in the `R/` subdirectory are automatically sourced when the
 #'   application is run.
-#' * `R/sort.R` and `R/my-module.R` are automatically sourced when
+#' * `R/example.R` and `R/example-module.R` are automatically sourced when
 #'   the application is run. The first contains a function `lexical_sort()`,
 #'   and the second contains code for a [Shiny module](moduleServer()) which
```

Comment:
`[Shiny module](moduleServer())` doesn't actually take me to the docs for the `moduleServer()` from the help file, when viewing it in the IDE. Could either link to the article https://shiny.rstudio.com/articles/modules.html or rephrase as "contains code for a Shiny module created with the [moduleServer()]" function which is ..."